For a given SNP column, the missing values will be replaces by sampling from the three possible genotype 
(in this case {0,1,2}, where: 0 stands for minor allele homozygous, 1 for hetrozygous and 2 for major allele homozygous) 
with the probalilities for each given by observed probalilities for that SNP. 
This function is meant to replace missing genotype (NA) calls in SNP genotype 
data after quality control measures such as MAF, call rate and genotype completion. 
It is not meant to impute genotype for SNP that do not exist in the given data.
The function takes the following 2 arguments:- 

1. Name of the genotype file which has to be in the working directory: 
    The file sould be generated by PLINK command "--recode A" with genotypes 
    coded as '0', '1', '2'.  Missing genotypes have to be coded as 'NA' 
    (which --recode A does by default).The function can be easily midified 
    for other kinds of genotype coding.

2. The file format of the output file which includes imputed genotypes. 
    Two options can be specified: "csv" for a coma delimited file, and 
                                  "space" for space delimited text file. 
